config/query/format_boosting.yml

```diff
@@ -0,0 +1,49 @@
+government_index:
```

Comment:
Good point. The long-term plan is to use document supertypes, which will remove the need for a hard-coded format list. I've added a comment to explain.